internal/test/envtest/environment.go

```diff
@@ -201,6 +214,10 @@ func (e *Environment) CreateNamespaceForTest(ctx context.Context, t *testing.T)
 }
 
 func getRootPath() string {
+	return path.Join(currentDir(), "..", "..", "..")
```

Comment:
Can this work instead of guessing the 4 dirs? https://stackoverflow.com/a/32163888

Reply:
That's what we use to get `currentDir()`. But from there, we need to go to the root of the repo. And we know that's 3 folders up the stack because we know where this file is.